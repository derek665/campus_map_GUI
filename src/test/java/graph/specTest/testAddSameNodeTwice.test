# test when adding a node already exists, nothing should be changed

# Create graph
CreateGraph dupl

# Add two nodes
AddNode dupl n1
AddNode dupl n2

# Add edge from n1
AddEdge dupl n1 n2 e1

# Print children of n1
ListChildren dupl n1

# Add a node
AddNode dupl n1

# Print children of n1
ListChildren dupl n1