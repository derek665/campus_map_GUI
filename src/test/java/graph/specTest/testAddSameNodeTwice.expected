# test when adding a node already exists, nothing should be changed

# Create graph
created graph dupl

# Add two nodes
added node n1 to dupl
added node n2 to dupl

# Add edge from n1
added edge e1 from n1 to n2 in dupl

# Print children of n1
the children of n1 in dupl are: e1

# Add node n1
# No change since n1 already exists

# Print children of n1
the children of n1 in dupl are: e1